### PR TITLE
[IGNORE] minor visual test improvements

### PR DESCRIPTION
**Pull request by juliepagano**
I caught a few minor issues with the visual tests while moving on to the more complicated case of creating tests for charts. I wanted to get these in first as a smaller, separate PR, and will follow-up with a PR for the chart tests later.

- Adjust check for loading to finish.
- Rename markdown test file to clarify it's for the panel.
- Use parent of panel to ensure the height/width are maintained.

Signed-off-by: Julie Pagano <julie.pagano@chronosphere.io>

# Checklist

- [x] Pull request has a descriptive title and context useful to a reviewer.
- [x] Pull request title follows the `[<catalog_entry>] <commit message>` naming convention using one of the following `catalog_entry` values: `FEATURE`, `ENHANCEMENT`, `BUGFIX`, `BREAKINGCHANGE`, `IGNORE`.
- [x] All commits have [DCO signoffs](https://github.com/probot/dco#how-it-works).
- [x] Changes that impact the UI include screenshots and/or screencasts of the relevant changes.

### CODE REVIEW

File: ui/e2e/src/pages/Panel.ts [hunk 1]
```diff
@@ -48,17 +53,18 @@ export class Panel {
     this.resizeHandle = this.container.locator('..').locator('.react-resizable-handle');
 
     this.figure = this.container.getByRole('figure');
-    this.loader = this.container.getByLabel('Loading');
+    this.loader = this.container.locator('[aria-label*=Loading]');
```
Comment (juliepagano) [inline ui/e2e/src/pages/Panel.ts:56]: This wasn't working quite how I expected -- I think it's focused on input labels. Moving to a more explicit locator.

File: ui/e2e/src/pages/Panel.ts [hunk 2]
```diff
@@ -48,17 +53,18 @@ export class Panel {
...
     // Wait for the figure to have at least one visible child that is not the loader.
     await expect(async () => {
-      expect(this.figure).not.toContain(this.loader);
+      await expect(this.loader).toHaveCount(0);
 
       const figureChildren = this.figure.locator('*:visible');
-
       expect(await figureChildren.count()).toBeGreaterThan(0);
+
+      await expect(this.loader).toHaveCount(0);
```
Comment (juliepagano) [inline ui/e2e/src/pages/Panel.ts:67]: Checking both before and after looking for children just to be extra sure and avoid a small edge case with timing that might lead to a flaky test. There might be some complex regex or something I could do with a single check, but this was a bit simpler to write.

File: ui/e2e/src/tests/markdownPanel.spec.ts [hunk 3]
```diff
@@ -34,7 +34,7 @@ test.describe('Dashboard: Markdown Panel', () => {
       const markdownPanel = dashboardPage.getPanel(panelName);
       await markdownPanel.isLoaded();
 
-      await happoPlaywright.screenshot(page, markdownPanel.container, {
+      await happoPlaywright.screenshot(page, markdownPanel.parent, {
```
Comment (juliepagano) [inline ui/e2e/src/tests/markdownPanel.spec.ts:37]: The width/height live on the container, so previously the screenshots were filling the whole page. This was way more noticeable when working on the canvas visualizations because happo turns them into images, so they weren't stretching. Moving to the parent, keeps them the size we intend to test.

Comment (juliepagano) [reply ui/e2e/src/tests/markdownPanel.spec.ts:38]: I swapped them to a 3 across size, so they're a little bigger now and probably at a common size people might use them in a dashboard. Does that look better to you?

Comment (cndonovan) [reply ui/e2e/src/tests/markdownPanel.spec.ts:38]: Oh where should I go to look, it looks like the Happo link above still points to old ones

Comment (juliepagano) [reply ui/e2e/src/tests/markdownPanel.spec.ts:38]: Oops, sorry about there. Here's the new one. https://happo.io/a/1009/p/1488/compare/9e577592bcdc75699bd55ad53ab0026e5e80ecf6/58c5e98ef29df87f73f159015136533ce65ea65c

You can also find it by clicking the "details" link on the happo check 
image

**Review by cndonovan — APPROVED** [review]

File: ui/e2e/src/tests/markdownPanel.spec.ts (hunk 3 above)
Comment (cndonovan) [reply ui/e2e/src/tests/markdownPanel.spec.ts:38]: Looks great, I like it!
